x-pack/legacy/plugins/lens/public/indexpattern_plugin/field_item.tsx

```diff
@@ -260,15 +271,15 @@ function FieldItemPopoverContents(props: State & FieldItemProps) {
   if (histogram && histogram.buckets.length && topValues && topValues.buckets.length) {
     title = (
       <EuiButtonGroup
-        buttonSize="s"
+        buttonSize="compressed"
```

Comment:
Think this is related? The label is being cut off for "Distribution" <img width="554" alt="Screenshot 2019-10-01 17 08 04" src="https://user-images.githubusercontent.com/666475/66000948-81662000-e46e-11e9-99ff-e71f032f8066.png">

Reply:
I don't understand why your screenshots keep showing up as all caps in those buttons. What env are you in?

Reply:
I primarily develop in Firefox

Reply:
That last commit should fix your issue.